lib/policy_machine.rb

```diff
@@ -127,7 +126,9 @@ def is_privilege_ignoring_prohibitions?(user_or_attribute, operation, object_or_
       raise(ArgumentError, "options[:associations] cannot be empty") if associations.empty?
       raise(ArgumentError, "expected each element of options[:associations] to be a PM::Association") unless associations.all?{|a| a.is_a?(PM::Association)}
 
-      associations.keep_if{ |assoc| assoc.includes_operation?(operation) }
+      associations.keep_if do |association|
+        association.operation_set.connected?(operation)
```

Comment:
Argh, it looks like this could use the AR storage adapter if associations are provided to the privilege call. That's not a thing we do, but it's a thing we _could_ do. I love the policy machine. I'll revisit this.